@LinProp/LinProp.m

```diff
@@ -431,161 +431,347 @@ function display(obj)
-                        dest_subs = cell(1, ni);
-                        for i = 1:ni
-                            if strcmp(src_subs{i}, ':')
-                                if ni == 1
-                                    src_subs{i} = 1:numel(A);
-                                else
-                                    src_subs{i} = 1:size(A, i);
-                                end
-                            end
-                            n(i) = numel(src_subs{i});
-                            dest_subs{i} = 1:n(i);
-                        end
-                        am = LinProp.Convert2UncArray(A);
-                        src_index  = LinProp.IndexMatrix(src_subs);
-                        dest_index = LinProp.IndexMatrix(dest_subs);
-                        if A.IsComplex
-                           bm = NET.createGeneric('Metas.UncLib.Core.Ndims.ComplexNArray', {'Metas.UncLib.LinProp.UncNumber'});
-                           bm.InitNd(int32(n(:)));
-                        else
-                           bm = NET.createGeneric('Metas.UncLib.Core.Ndims.RealNArray', {'Metas.UncLib.LinProp.UncNumber'});
-                           bm.InitNd(int32(n(:)));
-                        end
-                        if ni == 1
-                            bm.SetItems1d(int32(dest_index - 1), am.GetItems1d(int32(src_index - 1)));
-                        else
-                            bm.SetItemsNd(int32(dest_index - 1), am.GetItemsNd(int32(src_index - 1)));
-                        end
-                        B = LinProp.Convert2LinProp(bm);
-                        s = size(B);
-                        if length(s) > 2 && s(end) == 1
-                            B = reshape(B, s(1:end-1));
-                        end
+        %SUBSREF Subscripted reference.
+        %   A(I) is an array formed from the elements of A specified by the
+        %   subscript vector I.  The resulting array is the same size as I except
+        %   for the special case where A and I are both vectors.  In this case,
+        %   A(I) has the same number of elements as I but has the orientation of A.
+        %
+        %   A(I,J) is an array formed from the elements of the rectangular
+        %   submatrix of A specified by the subscript vectors I and J.  The
+        %   resulting array has LENGTH(I) rows and LENGTH(J) columns.  A colon used
+        %   as a subscript, as in A(I,:), indicates all columns of those rows
+        %   indicated by vector I. Similarly, A(:,J) = B means all rows of columns
+        %   J.
+        %
+        %   For multi-dimensional arrays, A(I,J,K,...) is the subarray specified by
+        %   the subscripts.  The result is LENGTH(I)-by-LENGTH(J)-by-LENGTH(K)-...
+        
+            if strcmp('.', {S.type})
+                error('Dot indexing is not supported for variables of this type.');   %TODO: This was allowed in  V2.4.7 of the uncLib MATLAB wrapper - but did not serve any function, as far as Dion can see.
```

Comment:
Dot indexing is not needed. And it was not working in V2.4.7.
So we can remove the TODO comment here.